reference/format/acl.rst

```diff
@@ -0,0 +1,105 @@
+Access Control Lists
+====================
+
++-----------+-----------------+
+| Filename  | ``acl.yml``     |
++-----------+-----------------+
+| Root Node | none            |
```

Comment:
in acl.yml file, root nodes is ACL resource id.

Please add example of siple acl.yml file to better understanding this file structure. For example:

```
oro_test_capability:
    label: oro.test.action.test_capability
    type: action
    group_name: ""
    bindings: ~

...
some_another_ACL_resource:
...
```